src/organization/configuration/identity-provider/org-settings-identity-provider.component.ts

```diff
@@ -39,6 +39,8 @@ export class OrgSettingsIdentityProviderComponent implements OnInit, AfterViewIn
 
   private unsubscribe$ = new Subject<boolean>();
 
+  private prviderKeys: string[] = [];
```

Comment:
Maybe to rename to something like:
```suggestion 
  private identityProviderFormControlKeys: string[] = [];
```